README.md

```diff
@@ -73,6 +73,14 @@ Run `yarn nx run <my-app>-e2e:e2e` to execute the e2e tests with [cypress](https
 
 Run `nx test my-app` to execute the unit tests with [Jest](https://jestjs.io), or `nx affected:test` to execute just unit tests affected by a change. You can also use `--watch` with these test to run jest in watch mode, see [Jest executor](https://nx.dev/packages/jest/executors/jest) for all CLI flags.
 
+#### Note: below only needed for trading-e2e
+
+To run tests locally using your own wallets you can add the following environment variables to `Cypress.json`
```

Comment:
chore (blocking): should bye cypress.json (lowercase)

Reply:
Also done